### PR TITLE
cmd: don't show the price if a snap has already been purchased

### PR DESCRIPTION
**Pull request by pete-woods**

### CODE REVIEW

**Comment by mvo5** [comment]
:+1: (although we probably want a test at some point)

**Comment by zyga** [comment]
Looks good, +1

**Comment by zyga** [comment]
Please merge it after tests are done

File: cmd/snap/cmd_find.go [hunk 1]
```diff
@@ -34,12 +34,17 @@ var longFindHelp = i18n.G(`
...
-func getPrice(prices map[string]float64, currency string) string {
+func getPrice(prices map[string]float64, currency, status string) string {
 	// If there are no prices, then the snap is free
 	if len(prices) == 0 {
 		return ""
 	}
 
+	// If the snap is priced, but has been purchased
+	if status == "available" {
+		return ""
```
Comment (chipaca) [inline cmd/snap/cmd_find.go]: and if instead you return `i18n.G("bought")`?

Comment (chipaca) [inline cmd/snap/cmd_find.go]: or purchased or paid for or whatever ;-)

**Comment by mvo5** [comment]
Nice! :+1:

**Comment by chipaca** [comment]
retest this please

**Comment by chipaca** [comment]
retest this please

**Comment by mvo5** [comment]
retest this please

**Pull request merged**